docs/examples/EdgeX_ExampleModbus_configuration.toml

```diff
@@ -4,7 +4,7 @@ Port = 49991
 ConnectRetries = 3
 Labels = []
 OpenMsg = "device modbus started"
-MaxResultCount = 50000
+ReadMaxLimit = 256
```

Comment:
see previous comment for mqtt example configuration.toml

Reply:
I've removed this line as recommended by @cloudxxx8